docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -465,29 +509,180 @@ name, and allow ``alice`` and ``bob`` to use a group principal named as
       ]
     }
 
-.. _system_information_rules:
+.. _system-file-auth-system_information:
 
 System information rules
 ------------------------
 
-These rules specify which users can access the system information management interface.
-The user is granted or denied access, based on the first matching rule read from top to
-bottom. If no rules are specified, all access to system information is denied. If
-no rule matches, system access is denied. Each rule is composed of the following fields:
+These rules specify which users can access the system information management
+interface. The user is granted or denied access, based on the first matching
+rule read from top to bottom. If no rules are specified, all access to system
+information is denied. If no rule matches, system access is denied. Each rule is
+composed of the following fields:
 
 * ``user`` (optional): regex to match against user name. If matched, it
-  will grant or deny the authorization based on the value of ``allow``.
-* ``allow`` (required): set of access permissions granted to user. Values: ``read``, ``write``
+  grants or denies the authorization based on the value of ``allow``.
+* ``allow`` (required): set of access permissions granted to user. Values:
+  ``read``, ``write``
 
 For example, if you want to allow only the role ``admin`` to read and write
-system information, allow ``alice`` to read system information, and deny all other access, you
-can use the following rules:
+system information, allow ``alice`` to read system information, and deny all
+other access, you can use the following rules:
 
 .. literalinclude:: system-information-access.json
     :language: json
 
 A fixed user can be set for management interfaces using the ``management.user``
-configuration property.  When this is configured, system information rules must still be set
-to authorize this user to read or write to management information. The fixed management user
-only applies to HTTP by default. To enable the fixed user over HTTPS, set the
-``management.user.https-enabled`` configuration property.
+configuration property.  When this is configured, system information rules must
+still be set to authorize this user to read or write to management information.
+The fixed management user only applies to HTTP by default. To enable the fixed
+user over HTTPS, set the ``management.user.https-enabled`` configuration
+property.
+
+.. _catalog-file-based-access-control:
+
+Catalog-level access control files
+==================================
+
+You can create JSON files for individual catalogs that define authorization
+rules specific to that catalog. In the catalog configuration file, you then
+add a configuration property that sets the authorization type to ``FILE`` and
+points to this configuration file. Catalog-level access control files are
+supported on a per-connector basis, refer to the connector documentation for
+more information.
```

Comment:
the intro for system level control is better structured, mentioning the configs that need to be set etc.

Reply:
Rewrote this catalog-level intro to be more helpful and in line with the system-level intro.